### DISCUSSION TITLE
Can I store string values as project or design variables?

### DISCUSSION THREAD

**Discussion by s-benter** [ansys/pyaedt#4292] [Q&A] [answered]
For my automation workflow, I want to save configuration data for my inputs as part of the project or design.  For example, I have an input csv to set thermal parameters for Icepak.  I want to store the path to that csv in the project or design for later access.

I tried VariableManager but it doesn't accept strings as variable values.  Is there anywhere to store such a value within the project file itself, or do I need to create a separate file to track this?

**Comment by gmalinve** [comment, marked as answer]
Hi @s-benter,

Thanks for opening this discussion.
I'm not sure I understood correctly what you are trying to achieve but I think PyAEDT configuration files will give you great benefit: [Load and manage files](https://aedt.docs.pyansys.com/version/stable/User_guide/files.html). With them you can import a set of variables, setup, mesh and boundaries to your project. 
Since I see you are interested in Icepak please look at this [page](https://aedt.docs.pyansys.com/version/stable/User_guide/pyaedt_file_data/project.html) and specifically download the available Icepak example, here you will find an example on how to import a configuration file.

If this helps please do let me know.

Thanks, 

Giulia
